### PR TITLE
`between` function to find if number is within a range of numbers

### PR DESCRIPTION
**Pull request by cglane**

### CODE REVIEW

File: packages/ramda-extension/src/between.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * @example
+ *
+ *      R_.between(4, 1, 5); //=> true
+ *      R_.between(2.1, 3, 8); //=> false
+ *      R_.between(100.1, 100.1, 102); //=> true
+
+ */
+
+
+const between = (val, a, b) => {
```
Comment (tommmyy) [inline packages/ramda-extension/src/between.js]: Function must be curried.

File: packages/ramda-extension/src/between.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	if (val >= a && val <= b) {
```
Comment (tommmyy) [inline packages/ramda-extension/src/between.js]: we use pointfree code

File: packages/ramda-extension/src/between.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ *      R_.between(4, 1, 5); //=> true
+ *      R_.between(2.1, 3, 8); //=> false
+ *      R_.between(100.1, 100.1, 102); //=> true
+ *      R_.between(100.1)(100.1, 102); //=> true
+
+
+ */
+
+
+const between = curry((val, a, b) => {
```
Comment (jbradle) [inline packages/ramda-extension/src/between.js]: functions should be written as that the data to be operated on comes last. So (a, b, val)

File: packages/ramda-extension/src/between.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { curry } from 'ramda';
+/**
+ * See if an number (`val`) is within an array of two numbers ('list').
```
Comment (jbradle) [inline packages/ramda-extension/src/between.js]: See if **_a_** number

**Comment by tommmyy** [comment]
See https://ramda-extension.firebaseapp.com/repl/?v=0.25.0&ve=0.3.2#?const%20between%20%3D%20useWith%28compose%28allPass%2C%20R_.argumentsToList%29%2C%20%5Blte%2C%20gte%5D%29%0A%0Aconsole.clear%28%29%0Aconsole.log%28%0Abetween%281%2C%203%29%280%29%2C%0Abetween%281%2C%203%29%284%29%2C%20%0Abetween%281%2C%203%29%282%29%20%0A%29

File: packages/ramda-extension/src/__tests__/between-test.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { between } from '../';
+
+describe('between', () => {
+	it('should return true between numbers', () => {
+		expect(between(1.5, .4, 10)).toEqual(true);
```
Comment (lukasduspiva) [inline packages/ramda-extension/src/__tests__/between-test.js]: Just a hint, when you expecting true/false values you should use toBeTruthy/toBeFalsy matchers from jest:
- https://facebook.github.io/jest/docs/en/expect.html#tobetruthy

File: packages/ramda-extension/src/between.js (hunk 3 above)
Comment (lukasduspiva) [reply packages/ramda-extension/src/between.js]: Maybe there should be better naming - like min, max, value.

Comment (tommmyy) [reply packages/ramda-extension/src/between.js]: @lukasduspiva not needed - see pointfree implementation bellow

File: packages/ramda-extension/src/__tests__/between-test.js (hunk 5 above)
Comment (tommmyy) [reply packages/ramda-extension/src/__tests__/between-test.js]: @lukasduspiva actually, toEqual(true) is better than toBeTruthy... toBeTruthy accepts all truthy values such as `1` or `{}`... but we want only `boolean`.. `so toEqual(true)` is far more descriptive

File: packages/ramda-extension/src/between.js [hunk 6]
```diff
@@ -10,17 +10,16 @@ import { curry } from 'ramda';
...
+ *      R_.between(3, 8, 2.1); //=> false
+ *      R_.between(100.1, 102, 100.1); //=> true
 
 
  */
 
 
-const between = curry((val, a, b) => {
-	if (val >= a && val <= b) {
+const between = curry((min, max, val) => {
```
Comment (tommmyy) [inline packages/ramda-extension/src/between.js]: This is not exactly pointfree implementation. Do you need help?
This is snippet mentioned before:
```js
const between = useWith(compose(allPass, R_.argumentsToList), [lte, gte])

console.log(
    between(1, 3)(0), // false
    between(1, 3)(4), // false
    between(1, 3)(2)  // true
)
```

**Pull request merged**